### PR TITLE
Block:change usb controller to usb3 for pseries

### PR DESCRIPTION
**Pull request by maxujun**
ID:1899393
Signed-off-by: Xujun Ma <xuma@redhat.com>

### CODE REVIEW

**Comment by maxujun** [comment]
@MiriamDeng Could you help review?this depends another patch in 1899393.

**Comment by MiriamDeng** [comment]
Could you also give test result ? Thanks.

**Comment by maxujun** [comment]
> 
> 
> Could you also give test result ? Thanks. @maxujun

(1/2) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.ppc64le.io-github-autotest-qemu.multi_disk_random_hotplug.all_types: PASS (1104.18 s)
 (2/2) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.ppc64le.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.all_types: PASS (1254.95 s)

**Comment by maxujun** [comment]
@qingwangrh Could you help test on x86 platform and confirm if have problem?

**Comment by qingwangrh** [comment]
python ConfigTest.py --testcase=multi_disk_random_hotplug..all_types --guestname=RHEL.8.4.0 --driveformat=virtio_scsi  --clone=no --machine=q35,i440fx
(1/4) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.all_types: PASS (298.66 s)
 (2/4) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.all_types.q35: PASS (732.63 s)
 (3/4) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.all_types: PASS (288.18 s)
 (4/4) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.all_types.q35: PASS (763.11 s)

LGTM

**Comment by maxujun** [comment]
@PaulYuuu Could you have a look?

**Comment by qunfang** [comment]
@maxujun  "ID：1899393" format is incorrect, probably you were using Chinese input for the ":" symbol.  Please update.

**Comment by maxujun** [comment]
> 
> 
> @maxujun "ID：1899393" format is incorrect, probably you were using Chinese input for the ":" symbol. Please update.

Updated

File: qemu/tests/cfg/multi_disk_random_hotplug.cfg [hunk 1]
```diff
@@ -51,9 +51,11 @@
     variants:
         - all_types:
             stg_params = "fmt:virtio,virtio_scsi,lsi_scsi,usb2"
-            pseries:
-                stg_params = "fmt:virtio,virtio_scsi,usb2"
             usbs += " ehci"
+            pseries:
+                stg_params = "fmt:virtio,virtio_scsi,usb3"
```
Comment (PaulYuuu) [inline qemu/tests/cfg/multi_disk_random_hotplug.cfg:56]: Does RHEL7 or old qemu versions also apply to this rule?

Comment (maxujun) [reply qemu/tests/cfg/multi_disk_random_hotplug.cfg]: Upstream qemu still have ehci controller for power platform，downstream qemu  have no this controller from qemu-kvm-rhev-2.12.0,so add change in internal config and just add usb3 fmt support on py file here.

Comment (PaulYuuu) [reply qemu/tests/cfg/multi_disk_random_hotplug.cfg]: OK, thank you.

**Pull request merged**